inst/include/interface/rcpp/rcpp_objects/rcpp_population.hpp

```diff
@@ -144,6 +146,13 @@ class PopulationInterface : public PopulationInterfaceBase {
     for (int i = 0; i < ages.size(); i++) {
       population->ages[i] = this->ages[i];
     }
+    for (int i = 0; i < proportion_female.size(); i++) {
+      population->proportion_female[i] = this->proportion_female[i];
+      if(estimate_prop_female){
+        info->RegisterParameter(population->proportion_female[i]);
+      }
```

Comment:
Suggestion: 
- Maybe line 150 could be tested by replacing the hard-coded value of 0.5 from the tests/gtest/test_population_B_and_SB.cpp (see [here](https://github.com/NOAA-FIMS/FIMS/blob/main/tests/gtest/test_population_B_and_SB.cpp#L16) and [here](https://github.com/NOAA-FIMS/FIMS/blob/main/tests/gtest/test_population_B_and_SB.cpp#L16)) with the random values set up in `tests/gtest/test_population_test_fixture.hpp`? 

- If we want to include the `estimate_prop_female` feature in this pull request, maybe we can add a C++ or R unit test to ensure the flag works. Then, we can file a separate issue to write an integration test later.

Reply:
Thanks! I updated both hardcoded values of proportion_female in the google test with the population vector and added a unit test on the population Rcpp interface